src/ripple/server/impl/Port.cpp

```diff
@@ -205,6 +205,29 @@ parse_Port (ParsedPort& port, Section const& section, std::ostream& log)
         }
     }
 
+    {
+        auto const result = section.find("send_queue_limit");
+        if (result.second)
+        {
+            try
+            {
+                port.ws_queue_limit =
+                    beast::lexicalCastThrow<std::uint16_t>(result.first);
+
+                // Queue must be greater than 0
```

Comment:
Individual message sizes can be quite large. An unlimited queue size could be present unbound memory growth that will exhaust server resources. A queue requirement beyond the default requires special consideration as its likely the clients are unable to keep up with the server. I advise caution even with a private server and a large queue. Disconnects with a queue size beyond a few hundred in size probably mean the problem lies elsewhere.